pandas/core/frame.py

```diff
@@ -6811,6 +6811,9 @@ def unstack(self, level=-1, fill_value=None):
         Name to use for the 'value' column.
     col_level : int or str, optional
         If columns are a MultiIndex then use this level to melt.
+    keep_index : bool, optional, default False
```

Comment:
this needs to be called ignore_index as we use this everywhere else, default True i think (as that is what we do now)